src/examples/java/io/nats/examples/chaosTestApp/OutputErrorListener.java

```diff
@@ -129,4 +130,9 @@ public void pullStatusError(Connection conn, JetStreamSubscription sub, Status s
     public void flowControlProcessed(Connection conn, JetStreamSubscription sub, String id, FlowControlSource source) {
         Output.controlMessage(this.id, supplyMessage("INFO flowControlProcessed", conn, null, sub, "FlowControlSource:", source));
     }
+
+    @Override
+    public void socketWriteTimeout(Connection conn) {
+        Output.controlMessage(this.id, supplyMessage("SEVERE socketWriteTimeout", conn, null, null));
+    }
```

Comment:
This was added since the class is an ErrorListener and socketWriteTimeout was added.